### PR TITLE
test: Run smoke tests from all solutions build

### PR DESCRIPTION
**Pull request by tippmar-nr**
Invokes the linux smoke test workflow from the all solutions build.

### CODE REVIEW

File: .github/workflows/run_linux_smoke_tests.yml [hunk 1]
```diff
@@ -48,7 +54,15 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Download Agent Home Folders
+      - name: Download Agent Home Folders (Call)
+        if: ${{ inputs.external_call }}
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
```
Comment (nrcventura) [inline .github/workflows/run_linux_smoke_tests.yml:59]: Should the 2 download home folder steps use the same action?

Comment (tippmar-nr) [reply .github/workflows/run_linux_smoke_tests.yml:59]: No, the first one `(Call)` is used when the workflow is invoked from all_solutions and is downloading an artifact from the "current" run. The other `(Dispatch)` is when running the workflow manually and passing the ID of some previous all_solutions run.

Comment (nrcventura) [reply .github/workflows/run_linux_smoke_tests.yml:59]: I meant they are using different action libraries. One comes from `actions/download-artifact` and the other comes from `dawidd6/action-download-artifact`.

Comment (tippmar-nr) [reply .github/workflows/run_linux_smoke_tests.yml:59]: Right, they're two different actions that do different things - one downloads from the current build, one downloads from any specified build. See https://github.com/dawidd6/action-download-artifact#download-workflow-artifact-github-action

**Review by nrcventura — APPROVED** [review]

**Pull request merged**